src/lib/optimization/config/computedStatsConfig.ts

```diff
@@ -23,155 +13,170 @@ export type ComputedStatsConfigBaseType = {
+  UNCONVERTIBLE_EHR_BUFF: { label: { ns: 'optimizerTab', key: `${keyPrefix}Unconvertible`, args: { stat: 'Effect Hit Rate' } } },
+  UNCONVERTIBLE_BE_BUFF: { label: { ns: 'optimizerTab', key: `${keyPrefix}Unconvertible`, args: { stat: 'Break Effect' } } },
+  UNCONVERTIBLE_OHB_BUFF: { label: { ns: 'optimizerTab', key: `${keyPrefix}Unconvertible`, args: { stat: 'Outgoing Healing Boost' } } },
+  UNCONVERTIBLE_RES_BUFF: { label: { ns: 'optimizerTab', key: `${keyPrefix}Unconvertible`, args: { stat: 'Effect RES' } } },
+  UNCONVERTIBLE_ERR_BUFF: { label: { ns: 'optimizerTab', key: `${keyPrefix}Unconvertible`, args: { stat: 'Energy Regeneration Rate' } } },
 
   // EHP
-  DMG_RED_MULTI: { default: 1, label: 'DMG reduction' }, // Dmg reduction multiplier for EHP calcs - this should be multiplied by (1 - multi) instead of additive
-  EHP: { flat: true, label: 'Effective HP' },
+  DMG_RED_MULTI: { default: 1, label: { ns: '', key: 'DMG reduction' } }, // Dmg reduction multiplier for EHP calcs - this should be multiplied by (1 - multi) instead of additive
+  EHP: { flat: true, label: { ns: '', key: 'Effective HP' } },
 
   // Misc configs
-  SUMMONS: { flat: true, label: 'Summons' },
-  MEMOSPRITE: { bool: true, label: 'Memosprite' },
-  ENEMY_WEAKNESS_BROKEN: { bool: true, label: 'Enemy weakness broken' },
-  MEMO_BUFF_PRIORITY: { bool: true, label: 'Prioritize memosprite buffs' },
-  DEPRIORITIZE_BUFFS: { bool: true, label: 'Deprioritize buffs' },
-  COMBO_DMG: { flat: true, label: 'Combo DMG' },
+  SUMMONS: { flat: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Summons` } },
+  MEMOSPRITE: { bool: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Memosprite` } },
+  ENEMY_WEAKNESS_BROKEN: { bool: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Enemy weakness broken` } },
+  MEMO_BUFF_PRIORITY: { bool: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Prioritize memosprite buffs` } },
+  DEPRIORITIZE_BUFFS: { bool: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Deprioritize buffs` } },
+  COMBO_DMG: { flat: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Combo DMG` } },
 
   // DOT
-  DOT_CHANCE: { label: 'Dot base chance' },
-  EFFECT_RES_PEN: { label: 'Effect RES PEN' },
-  DOT_SPLIT: { label: 'Dot DMG split' }, // Black Swan's stacking DoTs, the initial DoT has full value but subsequent stacks have reduced (DOT_SPLIT) value
-  DOT_STACKS: { flat: true, label: 'Dot stacks' },
+  DOT_CHANCE: { label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Dot base chance` } },
+  EFFECT_RES_PEN: { label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Effect RES PEN` } },
+  DOT_SPLIT: { label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Dot DMG split` } }, // Black Swan's stacking DoTs, the initial DoT has full value but subsequent stacks have reduced (DOT_SPLIT) value
+  DOT_STACKS: { flat: true, label: { ns: 'optimizerTab', key: `${keyPrefix}Misc.Dot stacks` } },
```

Comment:
When there's this much repeated code it should definitely be moved into helper functions

Reply:
I think these would be enough to cover all the stats

```ts
const optimizerTabMisc = (key: string) => ({ ns: 'optimizerTab', key: `ExpandedDataPanel.BuffsAnalysisDisplay.Stats.Misc.${labelId}` })
const optimizerTabUnconvertible = (stat: Stat) => ...
const optimizerTabElementalStat = (stat: Stat, key: string) => ...
const optimizerTabComposite = (key: string) => ...
const commonReadable = (stat: Stat) => ...
const commonElemental = (stat: Stat) => ...

```